resources/install/build.xml

```diff
@@ -2468,7 +2460,7 @@
                  value="$(PACKAGE_NAME)"/>
         <replace file="${debianize.dir}/rules"
                  token=" usr/lib/jni/"
-                 value=" usr/share/$(PACKAGE_NAME)/lib/native -Xslickless.jar -Xslick-runner.jar -X-slick.jar -X.svn"/>
+                 value=" usr/share/$(PACKAGE_NAME)/lib/native/"/>
```

Comment:
Why stop excluding the slick stuff?

Reply:
Cause it is already there and this is duplication:
https://github.com/jitsi/jitsi/blob/master/resources/install/debian/rules.tmpl#L22

Reply:
Not sure I get all this Debian crap 🤷. Hope we can build a proper package and get rid of it at some point.